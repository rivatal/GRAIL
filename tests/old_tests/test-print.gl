main() {
	print("String");
/*	printbool(true);
	printbool(false); 
	printint(1); */
}