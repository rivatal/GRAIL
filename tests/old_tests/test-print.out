String1
0
1
